src/Controls/tests/DeviceTests/Elements/VisualElementTreeTests.cs

```diff
@@ -91,5 +95,113 @@ public async Task GetVisualTreeElements()
 
 			});
 		}
+
+		[Fact]
+		public async Task FindPlatformViewInsideLayout()
+		{
+			SetupBuilder();
+			var button = new Button();
```

Comment:
Could we include a test validating the extensions using a ContentView?. This will be used mostly by 3rd party folks where the use of ContentView and also templates is frequent.

Reply:
Added